src/views/wait-for-others-to-sign.njk

```diff
@@ -0,0 +1,44 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/breadcrumbs/macro.njk' import govukBreadcrumbs %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+  <main class="govuk-main-wrapper">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <div class="interruption-card">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-1">You must ask the other directors or members to sign the application</h1>
```

Comment:
Please add id to header for ease of UI testing. 'interruption-card-header' should suffice